### PR TITLE
added features section

### PR DESCRIPTION
**Pull request by jcobert**
Just poking around. Added a bootstrap card section to the homepage and an associated _feature.scss file. Removed the deprecated node-sass from package.json. Set the navbar logo to link to homepage.

### CODE REVIEW

File: assets/css/_custom.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // Override bootstrap classes
 $theme-colors: (
   "primary": #003366,
+  "theme-blue": #003366,
```
Comment (jmagardino) [inline assets/css/_custom.scss:4]: Why add this new `theme_color` if primary and `theme_blue` have the same value?

File: assets/css/app.scss [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /* This file is for your main application css. */
 @import "custom";
 @import "~bootstrap/scss/bootstrap";
+@import "feature";
```
Comment (jmagardino) [inline assets/css/app.scss:4]: I wonder if we need to import a new file or can just leverage `custom` ? 🤔

File: lib/kg_web/templates/page/index.html.eex [hunk 3]
```diff
@@ -4,3 +4,62 @@
     <p>A place to buy all of your eggs and check out our farm camp!</p>
   </div>
 </section>
+
+<section id="services">
```
Comment (jmagardino) [inline lib/kg_web/templates/page/index.html.eex:8]: This all looks awesome, nice work.

**Review by jmagardino — COMMENTED** [review]
nice!...

one general thing, we don't have to track the `.DS_Store` files in git because they are specific to your development environment. There is a file called `.gitignore` that allows us to ignore files like this. This stack overflow article explains how that works a bit.
https://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository

Also, was getting this error compiling, although things are seemingly working... did you see this error at all?

```
ERROR in ./css/app.scss
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: Undefined variable.
  ╷
5 │         color: $theme-blue;
  │                ^^^^^^^^^^^
  ╵
  css/_feature.scss 5:16  @import
  /Users/joemagardino/Desktop/code/kg/assets/css/app.scss 4:9               root stylesheet
```

File: assets/css/app.scss (hunk 2 above)
Comment (jcobert) [reply assets/css/app.scss:6]: custom from my understanding is conventionally used to store scss variables for things like colors and media query breakpoints, not actual styling rules.

File: assets/css/_custom.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (jcobert) [reply assets/css/_custom.scss]: That css error should be gone now. I like keeping a separate palette and not override bootstrap. I just incorrectly included it in the bootstrap theme-colors list.

**Comment by jmagardino** [comment]
🔥

**Review by jmagardino — APPROVED** [review]
Looks great man. Go ahead and "squash and merge this"... it will squash all your changes into one single commit onto the master branch.

For a next little task to get you familiar with the codebase, why don't you try directing the "Place Order" button to actually redirect to the "egg order page"... if you need an example, the header should already do this.

**Pull request merged**